django/contrib/messages/storage/cookie.py

```diff
@@ -31,12 +36,19 @@ class MessageDecoder(json.JSONDecoder):
     Decode JSON that includes serialized ``Message`` instances.
     """
 
+    def create_message(self, *vars):
+        """creates message on the basis of encoded data"""
+        args = vars[:-1]
+        restrictions = vars[-1]
```

Comment:
I appreciate keyword-only arguments...! Makes sense, but it's not nice to look at.